csm/core/services/storage_capacity.py

```diff
@@ -62,7 +62,7 @@ def convert_to_format(value: int, unit: str ,round_off_value:int) -> Any:
 
         try:
             process = AsyncioSubprocess(const.FILESYSTEM_STAT_CMD)
-            stdout, stderr = await process.run()
+            stdout, stderr, rc = await process.run()
         except Exception as e:
             raise CsmInternalError(f"Error in command execution command : {e}")
         if not stdout:
```

Comment:
might want to check if  not stdout or rc != 0:

Reply:
this is storage capacity code hence I just changed the rc receiving and nothing else @udayan-y21  can you please check this ??

Reply:
we can keep it as it for now